tests/app/main/test_frameworks.py

```diff
@@ -4890,15 +4893,14 @@ def test_valid_framework_agreement_returned_updates_api_and_sends_confirmation_e
         )
 
         # Delcaration primaryContactEmail and current_user.email_address are different so expect two recipients
-        mandrill_send_email.assert_called_once_with(
-            mock.ANY,  # self
-            to_email_addresses=['email2@email.com', 'email@email.com'],
-            email_body=mock.ANY,
-            subject='Your G-Cloud 8 signature page has been received',
-            from_email_address='enquiries@digitalmarketplace.service.gov.uk',
-            from_name='Digital Marketplace Admin',
-            tags=['g-cloud-8-framework-agreement']
```

Comment:
It'd be great if we could assert the notify call args on at least one of these tests as well as the recipients, that way we could make sure we're sending the right personalisations (including the full url).

Reply:
Asserting the full url is a pain, because it depends on the stage, which is why I did the funny business in [these lines](https://github.com/alphagov/digitalmarketplace-supplier-frontend/pull/935/files#diff-860e899ae62cbab98fee8fa871bcc53bR4900)

Reply:
Testing the personalisation in more detail can be done though